Library/Formula/cmake.rb

```diff
@@ -45,12 +50,13 @@ class Cmake < Formula
     sha256 "a4ec1aff59b95a14b45eb2e23761a0179e98319da5a7eb76b56ea8cdc7b871c3"
   end
 
+  # Fix for older bash-completion versions.  This can be removed when 3.3.0 is
+  # released.
   patch do
-    # fix for older bash-completion versions
     # Use Github because upstream git changes sha and breaks from-source download.
     url "https://github.com/Kitware/CMake/commit/2ecf168f1909.diff"
     sha256 "d3f8cd71d0b6ce23a22c55145114012da916f2e42af71cbbad35090d0aeb4f68"
-  end
+  end if build.stable?
```

Comment:
I kind of strongly, deeply dislike this format of patching. Let's do a stable block instead.